spec/unit/bootstrap/bundler_spec.rb

```diff
@@ -125,6 +125,28 @@
           expect(subject).not_to include('--clean')
         end
       end
+
+      context 'with ecs_compatibility' do
+        let(:options) { { :update => 'logstash-integration-jdbc' } }
+
+        it "should allow to update dependencies" do
```

Comment:
nit: let's avoid using "should" in example docstrings. I know this codebase has a lot of these already, but I'd like to try to stop introducing them :)

> Do not write 'should' or 'should not' in the beginning of your example docstrings. The descriptions represent actual functionality, not what might be happening. Use the third person in the present tense.
> -- [RSpec Style Guide - "Should" in Example Docstrings](https://github.com/rubocop/rspec-style-guide#should-in-example-docstrings)

```suggestion
        it "also updates dependencies" do
```